support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -70,11 +83,15 @@ switch (countryGroupId) {
 		break;
 }
 
+const isSignedIn = !!get('GU_U');
+const countryId: IsoCountry =
+	CountryHelper.fromString(get('GU_country') ?? 'GB') ?? 'GB';
+
```

Comment:
This is stuff we would normally manage in state, but is static on page load, so this felt like the easiest way to do it.